ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextAnchorViewManager.java

```diff
@@ -42,16 +42,19 @@
     Spacing.ALL, Spacing.LEFT, Spacing.RIGHT, Spacing.TOP, Spacing.BOTTOM,
   };
   private static final String TAG = "ReactTextAnchorViewManager";
+  private int mNumberOfLines = ViewDefaults.NUMBER_OF_LINES;
 
   // maxLines can only be set in master view (block), doesn't really make sense to set in a span
   @ReactProp(name = ViewProps.NUMBER_OF_LINES, defaultInt = ViewDefaults.NUMBER_OF_LINES)
   public void setNumberOfLines(ReactTextView view, int numberOfLines) {
+    mNumberOfLines = numberOfLines;
     view.setNumberOfLines(numberOfLines);
   }
 
   @ReactProp(name = ViewProps.ELLIPSIZE_MODE)
   public void setEllipsizeMode(ReactTextView view, @Nullable String ellipsizeMode) {
-    if (ellipsizeMode == null || ellipsizeMode.equals("tail")) {
+    boolean useDefaultValue = mNumberOfLines > 1 && !ellipsizeMode.equals("clip");
```

Comment:
Actually, @fabriziobertoglio1987 can we retrieve NumberOfLines from the view?

View Managers shouldn't keep state